cumulusci/tasks/robotframework/debugger/ui.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import print_function
+import cmd
+import textwrap
+import sys
+import re
+import pdb
+from robot.libraries.BuiltIn import BuiltIn
+from cumulusci.cli.ui import CliTable
+from selenium.common.exceptions import InvalidSelectorException
+
+
+class DebuggerCli(cmd.Cmd, object):
+    intro = textwrap.dedent(
+        """\
+        Welcome to rdb, the Robot Framework debugger
+
+        Type help or ? to list commands.
+    """
+    )
+    prompt = "rdb> "
+
+    def __init__(self, listener):
+
+        # robot redirects sys.stdout, use the original handle
+        cmd.Cmd.__init__(self, stdout=sys.__stdout__)
+
+        self.listener = listener
+        self.builtin = BuiltIn()
+
+        # some simple aliases.
+        self.do_c = self.do_continue
+        self.do_l = self.do_locate_elements
```

Comment:
You make a good point. I was thinking about implementing a "show source" command, and it would be smart to name that command "list" so that it's similar to pdb.  

I'll think about coming up with a different alias, and/or a different name for the `locate_elements' command

Reply:
how about x (for eXamine or Xpath) or h (for highlight)

Reply:
I want to shy away from xpath since you can also use css paths, ids, etc.  I think I'll leave it as `locate_elements` since the argument should be a locator, and change the alias to `le`.